src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
 	$.widget("ui.igBaseEditor", {
 		options: {
 			/* type="string|number|null" Gets/Sets how the width of the control can be set.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					width : 200
+				});
+
+				//Get
+				var width = $(".selector").%%WidgetName%%("option", "width");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "width", 200);
+				```
 				string The widget width can be set in pixels (px) and percentage (%).
 				number The widget width can be set as a number in pixels.
 				null type="object" will stretch to fit data, if no other widths are defined.
 			*/
 			width: null,
 			/* type="string|number|null" Gets/Sets how the height of the control can be set.
```

Comment:
I don't think the option control "how" the height of the control is set. It controls the heightitself.